### DISCUSSION TITLE
请问有Cloudflare 的Zero Trust账号，该如何把这里WARP+的账号替换成自己的Zero Trust账号

### DISCUSSION THREAD

**Discussion by chptnt** [vvbbnn00/WARP-Clash-API#193] [Q&A]
如题。请教各位大佬

**Comment by ChengSir-CC** [comment]
APP -  右上角菜单  -  账户  -   登录到  Zero Trust

**Comment by leetown3113** [comment]
同问

**Comment by ChengSir-CC** [comment]
@vvbbnn00 
同问，求教大佬，Zero Trust账号怎么获取key？有什么命令可以查看当前剩余流量吗？

**Reply by MATAABB** [reply]
zero trust流量无限

**Comment by hahach** [comment]
哥们搞过，但是针对的不是现在这个版本，我一直停留在三四月份的一个版本的，不清楚后面的有没有影响。
zerotruth账户不需要刷流量，所以当时我改了脚本里面的一些功能，让他不自动刷流量了。

对于如何适用，首先你要会提取zerotruth的私钥。
方法在项目 https://github.com/rany2/warp.sh 这里看
获取了私钥后，只需要进入容器内，在/app/account/目录下的account.json 文件，把里面的公钥私钥换成你的zerotruth的就行了，其他的不用管

**Reply by ChengSir-CC** [reply]
呦西，你是个好哥们腿子
